CONTRIBUTING.md

```diff
@@ -40,11 +69,23 @@ guidelines prior to filing a bug report.
    as well, like whether this is a persistent issue, or if it only occurs
    some of the time.
 
+## Feature Requests
+
+Open an [issue][issues] with the following:
+
+* A short, descriptive title. Ideally, other community members should be able 
+   to get a good idea of the feature just from reading the title.
+* A detailed description of the the proposed feature. 
+    * Why it should be added to the SDK.
+    *  If possible, example code to illustrate how it should work.
+* Use Markdown to make the request easier to read;
+* If you intend to implement this feature, indicate that you'd like to the issue to be assigned to you.
```

Comment:
If we already have issue templates for these, do we need to reiterate in the README the expected content?

Reply:
Not necessarily, looking over other repos' contributing file this section isn't present. Removing section altogether.